core/templates/dev/head/i18n.js

```diff
@@ -75,8 +75,9 @@ oppia.config([
   var availableLanguageKeys = [];
   var availableLanguageKeysMap = {};
   for (var prop in GLOBALS.SUPPORTED_SITE_LANGUAGES) {
-    availableLanguageKeys.push(prop);
-    availableLanguageKeysMap[prop + '*'] = prop;
+    availableLanguageKeys.push(GLOBALS.SUPPORTED_SITE_LANGUAGES[prop].id);
```

Comment:
No worries -- this was a bit more involved than I expected, too!

The fix is now in develop, so you should be able to merge and update your branch now.